mackup/applications/illustrator.cfg

```diff
@@ -11,3 +11,20 @@ Library/Preferences/Adobe Illustrator 18 Settings
 Library/Preferences/Adobe Illustrator 19 Settings
 Library/Preferences/Adobe Illustrator 20 Settings
 Library/Preferences/com.adobe.illustrator.plist
+
```

Comment:
Not sure about empty lines in those files, let's just not add them.

Reply:
Agreed!

Reply:
Can you fix those if you agree? =)